tests/test_wave_03.py

```diff
@@ -54,13 +51,8 @@ def test_friends_unique_movies_not_duplicated():
 
     # Assert
     assert len(friends_unique_movies) == 3
+    assert friends_unique_movies == [FANTASY_4, HORROR_1, INTRIGUE_3]
```

Comment:
[nit] I'd probably assert each individually, similar to `test_friends_unique_movies`. Asserting equality with the list forces a specific *order* as well.